sympy/physics/vector/frame.py

```diff
@@ -130,7 +130,8 @@ def __init__(self, name, indices=None, latexs=None, variables=None):
         >>> vlatex(P.x)
         'A1'
 
-        symbols() can be used to create multiple Reference Frames in one step, for example:
+        ``symbols()`` can be used to create multiple Reference Frames in one
```

Comment:
Even better is probably `:func:~.symbols` (with correct backticks around `~.symbols`, but not sure how to get those here...)

Reply:
I'll work on those in other PRs.